### PR TITLE
Add series endpoint

### PR DESCRIPTION
**Pull request by jurgenwerk**

### CODE REVIEW

File: source/endpoints/series.md.erb [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+]
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/series?access_token=ACCESS_TOKEN`
+
+Parameter | Description
+--------- | -----------
+url_ids   | Array of URL ids
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb]: `url_ids` is mandatory.

File: source/endpoints/series.md.erb [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+]
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/series?access_token=ACCESS_TOKEN`
+
+Parameter | Description
+--------- | -----------
+keyword_ids  | Array of keyword ids
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb]: `keyword_ids` is mandatory.

File: source/endpoints/series.md.erb [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/series?access_token=ACCESS_TOKEN`
+
+Parameter | Description
+--------- | -----------
+url_ids   | Array of URL ids
+with_competitors | Include competititor URL series (true or false, default: true)
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb:51]: We name this option but we don't show an example, I think we should include a brief mention as they resemble a lot to URL series anyway.

File: source/endpoints/series.md.erb [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+Available keyword series:
+
+Series | Description
+--------- | -----------
+average_position    | Average rank position
+search_visibility_index   | Search visibility index ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+click_potential | Click potential ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+keyword_distribution | Distribution of keyword positions in Top 3, Top 10, Top 100, No rank
+total_keywords | Number of keywords that fall into view's filters on specified days
+
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb:177]: `up_down` series is missing.

File: source/endpoints/series.md.erb [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+Available keyword series:
+
+Series | Description
+--------- | -----------
+average_position    | Average rank position
+search_visibility_index   | Search visibility index ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+click_potential | Click potential ([more info](https://docs.nightwatch.io/article/51-metrics-explanation))
+keyword_distribution | Distribution of keyword positions in Top 3, Top 10, Top 100, No rank
+total_keywords | Number of keywords in the group on specified days
+
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb:219]: `up_down` is missing.

File: source/endpoints/series.md.erb [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+Available keyword series:
+
+Series | Description
+--------- | -----------
+position  | Keyword's rank
+position_organic | Keyword's organic rank
+position_local_pack | Keyword's local pack rank
+position_places | Keywords rank in Google Places (when engine is Google Places)
+position_places_image | Keywords rank in image carousel (when engine is Google)
+
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb:135]: Google Analytics series are missing:
 - `clicks`
 - `impressions`
 - `search_console_position`
 - `ctr`

Might be worth noting that if GA integration is not setup these series will be empty (`[]`). Same notice applies to URL series.

File: source/endpoints/series.md.erb [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+  keyword_id: 1212189,
+  keyword: "goji berries",
+  keyword_series: [
+    {
+      name: "position",
+      series:  [["2019-04-22", 3], ...]
+    }, ...
+  ],
+  competitor_series: [
+    ...
```
Comment (emnlvrz) [inline source/endpoints/series.md.erb]: Maybe a word on competitor series?

**Review by emnlvrz — CHANGES_REQUESTED** [review]

File: source/endpoints/series.md.erb (hunk 2 above)
Comment (jurgenwerk) [reply source/endpoints/series.md.erb]: Added the asterisk.

File: source/endpoints/series.md.erb (hunk 3 above)
Comment (jurgenwerk) [reply source/endpoints/series.md.erb:51]: Added the example.

File: source/endpoints/series.md.erb (hunk 5 above)
Comment (jurgenwerk) [reply source/endpoints/series.md.erb:219]: Added.

File: source/endpoints/series.md.erb (hunk 7 above)
Comment (jurgenwerk) [reply source/endpoints/series.md.erb]: Added an example.

**Review by emnlvrz — APPROVED** [review]

**Pull request merged**